README.md

```diff
@@ -1,32 +1,31 @@
 # Pelias
 
-> A modular, open-source geocoder built on top of ElasticSearch for fast geocoding.
-
-[![Gitter](https://badges.gitter.im/pelias/pelias.svg)](https://gitter.im/pelias/pelias?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+> A modular, open-source geocoder built on top of Elasticsearch for fast and accurate global search.
 
 ## Announcement: See our [update](https://github.com/pelias/pelias/tree/master/announcements/2018-01-02-pelias-update/) regarding the Mapzen shutdown
 
 ### What's a geocoder do anyway?
 
-Geocoding is the process of transforming input text, such as an address, or a name of a place—to a location on the earth's surface.
+Geocoding is the process of transforming input text, such as an address, or a name of a place—to a location on the Earth's surface.
```

Comment:
the place-to part is a bit strange. Perhaps `Geocoding is the process of transforming input text, such as an address, or a name of a place, into a coordinate location (usually in latitude / longitude terms) on the Earth's surface.` That would also deal with "well, an address is a _location_"

Reply:
Good catch on both points.